sources/documents/RTB/Verbali/Interni/VerbaleInterno_2024-04-10/local.tex

```diff
@@ -0,0 +1,26 @@
+%%%%%%%%%%%%%%
+%  COSTANTI  %
+%%%%%%%%%%%%%%
+
+% In questa prima parte vanno definite le 'costanti' utilizzate soltanto da questo documento.
+% Devono iniziare con una lettera maiuscola per distinguersi dalle funzioni.
+
+\newcommand{\DocTitle}{Verbale Riunione 2024-04-10}
+\newcommand{\DocVersion}{1.0.0}
```

Comment:
Sul registro delle modifiche è indicata la versione 0.0.1, mentre a pié di pagina è indicata la 1.0.0. Sarebbe da modificare il comando in `local.tex` per poi usarlo nella versione corrente. Altrimenti si può lasciare così e cambiare solo in caso si vada sopra la versione 1.0.0

Reply:
Sì, in `local.tex` va messo 0.0.1, poi con il commit successivo si avanza di versione

Reply:
E @RaulPianon controlla anche il file relativo al registro delle modifiche, che vedendo il pdf mi sembra la data non sia corretta